service/controller/chart/v1/key/key.go

```diff
@@ -11,6 +11,8 @@ import (
 )
 
 const (
+	ChartOperatorName = "chart-operator"
+
 	// ForceHelmUpgradeAnnotationName is the name of the annotation that
 	// controls whether force is used when upgrading the Helm release.
 	ForceHelmUpgradeAnnotationName = "chart-operator.giantswarm.io/force-helm-upgrade"
```

Comment:
Sidenote, we have label and annotation packages in other projects now. This can maybe be improved in a separate PR.